activesupport/lib/active_support/core_ext/module/delegation.rb

```diff
@@ -273,6 +273,8 @@ def respond_to_missing?(name, include_private = false)
       def method_missing(method, *args, &block)
         if #{target}.respond_to?(method)
           #{target}.public_send(method, *args, &block)
+        elsif #{target}.nil?
+          raise DelegationError, "\#{method} delegated to #{target}, but #{target} is nil"
```

Comment:
That assumes `super` will raise; the class may well have other `method_missing` definitions in play.